modules/catalog-compaction/src/test/java/org/apache/ignite/internal/catalog/compaction/CatalogManagerCompactionFacadeTest.java

```diff
@@ -53,65 +46,6 @@ void setupHelper() {
         catalogManagerFacade = new CatalogManagerCompactionFacade(catalogManager);
     }
 
-    // TODO https://issues.apache.org/jira/browse/IGNITE-22522 Remove this test.
-    @Test
-    void testCollectTablesWithPartitionsBetween() {
-        CreateTableCommandBuilder tableCmdBuilder = CreateTableCommand.builder()
-                .schemaName("PUBLIC")
-                .columns(List.of(columnParams("key1", INT32), columnParams("key2", INT32), columnParams("val", INT32, true)))
-                .primaryKey(TableHashPrimaryKey.builder().columns(List.of("key1", "key2")).build())
-                .colocationColumns(List.of("key2"));
-
-        DropTableCommandBuilder dropTableCommandBuilder = DropTableCommand.builder()
-                        .schemaName("PUBLIC");
-
-        long from1 = clockService.nowLong();
-
-        assertThat(catalogManager.execute(tableCmdBuilder.tableName("test1").build()), willCompleteSuccessfully());
-        assertThat(catalogManager.execute(dropTableCommandBuilder.tableName("test1").build()), willCompleteSuccessfully());
-
-        long from2 = clockService.nowLong();
-
-        assertThat(catalogManager.execute(tableCmdBuilder.tableName("test2").build()), willCompleteSuccessfully());
-        assertThat(catalogManager.execute(dropTableCommandBuilder.tableName("test2").build()), willCompleteSuccessfully());
-
-        long from3 = clockService.nowLong();
-        assertThat(catalogManager.execute(tableCmdBuilder.tableName("test3").build()), willCompleteSuccessfully());
-        assertThat(catalogManager.execute(dropTableCommandBuilder.tableName("test3").build()), willCompleteSuccessfully());
-
-        {
-            Int2IntMap tablesWithParts = catalogManagerFacade.collectTablesWithPartitionsBetween(
-                    from1,
-                    clockService.nowLong());
-
-            assertThat(tablesWithParts.keySet(), hasSize(3));
-        }
-
-        {
-            Int2IntMap tablesWithParts = catalogManagerFacade.collectTablesWithPartitionsBetween(
-                    from2,
-                    clockService.nowLong());
-
-            assertThat(tablesWithParts.keySet(), hasSize(2));
-        }
-
-        {
-            Int2IntMap tablesWithParts = catalogManagerFacade.collectTablesWithPartitionsBetween(
-                    from3,
-                    clockService.nowLong());
-
-            assertThat(tablesWithParts.keySet(), hasSize(1));
-        }
-
-        {
-            Int2IntMap tablesWithParts = catalogManagerFacade.collectTablesWithPartitionsBetween(
-                    clockService.nowLong(),
-                    clockService.nowLong());
-
-            assertThat(tablesWithParts.keySet(), hasSize(0));
-        }
-    }
-
     @Test
     void testCollectZonesWithPartitionsBetween() {
```

Comment:
Not related to your change. There's a typo in `Int2IntMap tablesWithParts = catalogManagerFacade.collectZonesWithPartitionsBetween(` and similar. **zones**WithParts should be used instead. Could you please fix it?

Reply:
Fixed.